DEVELOPMENT.md

```diff
@@ -275,48 +275,66 @@ Kubernetes service calls get redirected to the process on your local. Similarly
 the calls on the local process goes to actual services that are running in
 Kubernetes.
 
-- Install Telepresence
+### Prerequisites
 
+- Install Telepresence v2 (see the [installation instructions](https://www.getambassador.io/docs/telepresence/latest/install/) for details).
 - Deploy Knative Eventing on your Kubernetes cluster.
 
-- Install [EnvFile plugin](https://plugins.jetbrains.com/plugin/7861-envfile) to
-  your Intellij
+### Connect Telepresence and intercept the controller
 
-- Run following command to swap the controller with the controller that we will
-  start later.
+As a first step Telepresence needs to your Kubernetes cluster:
 
 ```
-telepresence --namespace knative-eventing --swap-deployment eventing-controller --env-json eventing-controller-local-env.json
+telepresence connect
 ```
 
-For debugging applications that receive traffic, such as webhooks, you also need
-to pass `--expose` parameter.
+_Hint: If this is your first time Telepresence connects to your cluster, you
+need to install the traffic manager too_
 
-For example:
+```
+telepresence helm install
+```
+
+As Telepresence v2 [needs a
+service](https://www.getambassador.io/docs/telepresence/latest/howtos/intercepts/#intercept-a-service-in-your-own-environment)
+in front of your planned intercepted component (e.g. the controller), you need
+to add a Kubernetes service for your component. E.g.:
 
 ```
-telepresence --swap-deployment kafka-controller-manager --namespace knative-eventing --env-json kafka-controller-manager.json --expose 8443
+kubectl -n knative-eventing expose deploy/eventing-controller
+```
+
+Afterwards you can run the following command to swap the controller with the
+local controller that we will start later.
+
+```
+telepresence intercept eventing-controller --namespace knative-eventing --port 8080:8080 --env-file ./eventing-controller.env
 ```
 
 This will replace the `eventing-controller` deployment on the cluster with a
```

Comment:
I am seeing the pod having a second contain, but it does not come up:

```
  Normal   Created    2m20s                  kubelet            Created container traffic-agent
  Normal   Started    2m20s                  kubelet            Started container traffic-agent
  Normal   Pulled     2m20s                  kubelet            Container image "docker.io/datawire/ambassador-telepresence-agent:1.13.3" already present on machine
  Warning  Unhealthy  2m17s (x4 over 2m19s)  kubelet            Readiness probe failed: stat: can't stat '/tmp/agent/ready': No such file or directory
  Normal   Started    98s (x2 over 2m20s)    kubelet            Started container eventing-controller
  Normal   Created    98s (x2 over 2m20s)    kubelet            Created container eventing-controller
  Normal   Pulled     98s (x2 over 2m20s)    kubelet            Container image "docker.io/matzew/controller-17550e04ebf9af897aa4b799ad074e13@sha256:6231ad6b64e0635fc7691498a3065a56f747f125e3b68a41fb45a6801c67ba1e" already present on machine
  Warning  Unhealthy  98s (x4 over 118s)     kubelet            Readiness probe failed: Get "http://10.244.0.8:8080/readiness": EOF
  Warning  Unhealthy  98s (x3 over 118s)     kubelet            Liveness probe failed: Get "http://10.244.0.8:8080/health": EOF
  Normal   Killing    98s                    kubelet            Container eventing-controller failed liveness probe, will be restarted
```

Reply:
As soon as I connected the debugger from my IDE (VSCode in my case) and the controller started that way, both container became ready. Did you "start" the debugger and so trigger a start of the controller?

Reply:
I see the debugs now in VS code.